### DISCUSSION TITLE
Summary view for the repository located in the working directory and the one linked to the current buffer

### DISCUSSION THREAD

**Discussion by iamFIREcracker** [tpope/vim-fugitive#2209] [Q&A] [answered]
Sometimes I just want to pop open the summary view and have it show information about the repository in the _current working directory_ and not the one linked to the current buffer.  Is there an idiomatic way to achieve that?

I am currently using: `:edit .git/index`, and I am quite happy with it; however, fugitive would spit the following warning, and I am curious to understand if there is a repacement for that, or if the workflow was discouraged altogether.

Here is the warning:

> fugitive: Direct editing of .git/index is deprecated

**Comment by tpope** [comment, marked as answer]
The thing that is changing is the status buffer is moving from `.git/index` to a `fugitive://` URL. URLs aren't relative to the current working directory, so there's no drop in replacement. Best you could do is wrap it in a command:

```vim
command! -bang -bar GCWDStatus
      \ if empty(FugitiveExtractGitDir(getcwd())) |
      \   echoerr 'Not in a Git repository' |
      \ else |
      \   exe 'edit<bang>' fnameescape(FugitiveFind(':', FugitiveExtractGitDir(getcwd()))) |
      \ endif
```

**Reply by iamFIREcracker** [reply]
I don't mind custom commands, so I am happy dropping that in my vimrc.  Thanks lots!
